conozco.py

```diff
@@ -1047,7 +1047,7 @@ def load_stats(self):
                         val = f.readline()
                         val = val.strip('\n')
                         if not(val == ''):
-                            l[i] = int(val)
+                            l[i] = int(float(val))
```

Comment:
Can you share the traceback this fixes? I don't see a reason why you'll convert to `float` then `int`.

Reply:
@ibiam In your comment https://github.com/sugarlabs/iknowSriLanka/pull/4#issuecomment-626194653 you had mentioned of an error

```bash
Cannot load stats invalid literal for int() with base 10: '0.0'
```

Small reproducer
```python
Python 3.8.2 (default, Apr  8 2020, 14:31:25) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int('5.555')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '5.555'
>>> float('5.5555')
5.5555
>>> int(float('5.5555'))
5
>>> 
```

This was the reason behind the fix

Reply:
Yeah, thanks for that.